tests/ui/parser/issue-112188.rs

```diff
@@ -0,0 +1,13 @@
+#![allow(unused)]
```

Comment:
Can you use `// run-rustfix` on this?

Reply:
I removed the `Foo { .., .. }` case since the suggestion diagnostics currently doesn't handle that.